tester/src/test/scala/spinal/tester/scalatest/SpinalSimWishboneAdapterTester.scala

```diff
@@ -14,28 +14,32 @@ class WishboneSimpleBusAdapted( configIn : WishboneConfig,
                                 configOut : WishboneConfig,
                                 allowAddressResize : Boolean = false,
                                 allowDataResize : Boolean = false,
-                                allowTagResize : Boolean = false) extends Component{
+                                allowTagResize : Boolean = false,
+                                autoconnect : Boolean = false) extends Component{
   val io = new Bundle{
     val busIN = slave(Wishbone(configIn))
     val busOUT = master(Wishbone(configOut))
   }
-  val ff = Reg(Bool())
-  val adapter = WishboneAdapter(io.busIN,io.busOUT,allowAddressResize,allowDataResize,allowTagResize)
+  if(autoconnect) new Area {
```

Comment:
I think I was trying to get those symbols to show up in gtkwave; I don't recall if that actually worked though to be honest. I can take it out

Reply:
If you don't assign the area to a variable it won't help since there is then no name for the area to be inferred - though `adapter` should get a name either way...
I'd say assign it to a variable if you want to keep it so it gets a name or remove it - but it's not a big deal, I mostly asked just to understand